src/QirRuntime/test/QIR-static/compiler/QirCore.qs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
At some point in the future (when there is no more need for manual edits in the .ll files) we should be able to replace the QirCore.qs and QirTarget.qs files with a project reference to M.Q.QSharp.Core, and generate the .ll files as part of the tests.

Reply:
That would be great :)